WSL/install.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Install WSL
+description: Install Windows Subsystem for Linux with the command, wsl --install. Use a Bash terminal on your Windows machine run by your preferred Linux distribution - Ubuntu, Debian, SUSE, Kali, Fedora, Pengwin, Alpine, and more are available.
+keywords: wsl, install, BashOnWindows, bash, windows subsystem for linux, install ubuntu on windows, enable WSL2, linux on windows
+ms.date: 09/10/2021
+ms.topic: article
+ms.localizationpriority: high
+---
+
+# Install WSL
+
+You can now install everything you need to run Windows Subsystem for Linux (WSL) by entering this command in PowerShell or Windows Command Prompt and then restarting your machine.
+
+```powershell
+wsl --install
+```
+
+> [!IMPORTANT]
+> The `wsl --install` command is only supported on Windows 10 version 2004 and higher (Build 19041 and higher) or Windows 11. If you're running an older build, or just prefer not to use the install command and would like step-by-step directions, see **[WSL manual installation steps for older versions](./install-manual.md)**.
+>
+> To check your Windows version and build number, select **Windows logo key + R**, type **winver**, select **OK**. You can [update to the latest Windows version](ms-settings:windowsupdate) in the Settings menu or using the [Windows Update Assistant](https://www.microsoft.com/software-download/windows10).
+
+
+## Understanding the install command
```

Comment:
TBH I think we could remove this section and just add a paragraph under the `# Install WSL` section . 

I picture something like:

```
## Install WSL 

You can now install everything you need to run Windows Subsystem for Linux (WSL) by entering this command in PowerShell or Windows Command Prompt and then restarting your machine.

powershell
wsl --install

This command will enable all the required optional components, set you up to use WSL 2 distros as your default, and install a Linux distribution for you *(Ubuntu by default, see below to change this)*. 

```

I think that this will help the doc be more concise.